### PR TITLE
[Homework 04] - CSS by Bruno Diaz

### PR DESCRIPTION
**Pull request by BrunoDiaz7**
@VitalikMelnyk Hi! I've made some corrections to the structure of my document and the organization of my files

### CODE REVIEW

File: homeworks/bruno-diaz/04-CSS/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
```
Comment (VitalikMelnyk) [inline homeworks/bruno-diaz/04-CSS/index.html:1]: Please, review and rewrite BEM approach.
See the official docs for BEM and some cheatsheet for quick understanding.
https://bem-cheat-sheet.9elements.com/

File: homeworks/bruno-diaz/04-CSS/light.css [hunk 2]
```diff
@@ -0,0 +1,287 @@
+body {
```
Comment (VitalikMelnyk) [inline homeworks/bruno-diaz/04-CSS/light.css:1]: You have an annoying horizontal scroll.
Please, review your styles.
Screenshot 2023-07-17 at 13 10 55

Comment (VitalikMelnyk) [inline homeworks/bruno-diaz/04-CSS/light.css:1]: Secondly, when you toggle the theme, your spacing inside the portfolio card sections is different.
Please, review this moment.

**Comment by BrunoDiaz7** [comment]
@VitalikMelnyk Hi! I think I've solved the problems in my homework. I would appreciate it if you could review the changes

**Review by solvd-mmelnyk — APPROVED** [review]

**Pull request merged**